docs/index.md

```diff
@@ -1295,6 +1298,69 @@ The "HTML" reporter is what you see when running Mocha in the browser.  It looks
 
 [Mochawesome](https://www.npmjs.com/package/mochawesome) is a great alternative to the default HTML reporter.
 
+## Desktop Notification Support
+
+Desktop notifications allow asynchronous communication of events without
+forcing you to react to a notification immediately. Their appearance
+and specific functionality vary across platforms. They typically disappear
+automatically after a short delay, but their content is often stored in some
+manner that allows you to access past notifications.
+
+[Growl][] was an early notification system implementation for OS X and Windows,
+hence, the name of Mocha's `--growl` option.
+
+Once enabled, when your root suite completes test execution, a desktop
+notification should appear informing you whether your tests passed or failed.
+
+### Node-based notifications
+
+In order to use desktop notifications with the command-line interface (CLI),
+you **must** first install some platform-specific prerequisite software.
+Instructions for doing so can be found [here](https://github.com/mochajs/mocha/wiki/Growl-Notifications).
+
+Enable Mocha's desktop notifications as follows:
+
+```bash
+$ mocha --growl
+```
+
+### Browser-based notifications
+
+Web notification support is being made available for current versions of
+modern browsers. Ensure your browser version supports both
+[promises](https://caniuse.com/#feat=promises) and
+[web notifications](https://caniuse.com/#feat=notifications). As the
+Notification API evolved over time, **do not expect** the minimum possible
+browser version to necessarily work.
+
+Enable Mocha's web notifications with a slight modification to your
+client-side mocha HTML. Add a call to `mocha.growl()` prior to running your
+tests as shown below:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Mocha</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://unpkg.com/mocha@5.3.0/mocha.css" />
+  </head>
+  <body>
+    <div id="mocha"></div>
+    <script src="https://unpkg.com/mocha@5.3.0/mocha.js"></script>
```

Comment:
weird mocha version?

Reply:
Had assumed a minor release was next when written. You want `6.0.0`?

Reply:
ah, yeah, I think the next will be v6.0.0, sorry I wasn't clear.  if you feel like we need a minor before then, then we could cherry-pick our way to into it, but I'd rather just cut `master`

Reply:
Think we _should_ have but can't now... The engine specification changed long ago.

Reply:
Done